src/main/kotlin/racingcar/util/RandomGenerator.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar.util
+
+interface RandomGenerator {
+    fun generate(range: IntRange): Int
```

Comment:
랜덤영역을 인터페이스로 분리해주셨네요 👍 

사실 인터페이스의 관심사는 Number Generator 는 아닐까요?
Random 관련 관심사는 구현체에게 주어도 좋을거같아요 :)
```suggestion
interface NumberGenerator {
    fun generate(range: IntRange): Int
```

Reply:
넵 인터페이스의 이름을 포괄적으로 구현체 이름을 세분화 시켜보겠습니다! 👍